.github/actions/setup_proto/action.yaml

```diff
@@ -0,0 +1,9 @@
+name: Setup Ruby
+description: Setup Ruby
+
+runs:
+  using: composite
+  steps:
+    - name: Install cmake
+      run: |
+        sudo apt-get install cmake
```

Comment:
Protoc requires `cmake`. These setup actions could be resolved by pulling in a build system.